### PR TITLE
changelog: v0.3.2

### PR DESCRIPTION
**Pull request by webern**
**Issue number:**

N/A

**Description of changes:**

Update changelog for `v0.3.2`

**Testing done:**

N/A

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,98 @@
...
+* Bump cargo-make to 0.30.0 ([#870])
+* Pass proxy environment variables through to docker containers. ([#871])
+* Add parse-datetime crate ([#875])
+* Third-party software package version bumps ([#895])
+  * libseccomp: bump version from `v2.4.2` to `v2.4.3` ([403bdeab])
+  * conntrack-tools: bump version from `v1.4.5` to `v1.4.6` ([06b9a942])
+  * libnftnl: bump version from `v1.1.5` to `v1.1.6` ([dfc5b5da])
+  * libnetfilter_conntrack: bump version from `v1.0.7` to `v1.0.8` ([54771d69])
+  * libxcrypt: bump version from `v4.4.15` to `v4.4.16` ([979249e5])
+  * strace: version bump from `v5.5` to `v5.6` ([63ddb075])
```
Comment (tjkirch) [inline CHANGELOG.md]:
```suggestion
* Update third-party software packages ([#895])
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,98 @@
...
+[#898]: https://github.com/bottlerocket-os/bottlerocket/pull/898
+[#899]: https://github.com/bottlerocket-os/bottlerocket/pull/899
+[#902]: https://github.com/bottlerocket-os/bottlerocket/pull/902
+[#903]: https://github.com/bottlerocket-os/bottlerocket/pull/903
+[06b9a942]: https://github.com/bottlerocket-os/bottlerocket/commit/06b9a942
+[403bdeab]: https://github.com/bottlerocket-os/bottlerocket/commit/403bdeab
+[54771d69]: https://github.com/bottlerocket-os/bottlerocket/commit/54771d69
+[63ddb075]: https://github.com/bottlerocket-os/bottlerocket/commit/63ddb075
+[979249e5]: https://github.com/bottlerocket-os/bottlerocket/commit/979249e5
+[dfc5b5da]: https://github.com/bottlerocket-os/bottlerocket/commit/dfc5b5da
```
Comment (tjkirch) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,98 @@
+# v0.3.2 (unreleased)
+
+## OS changes
+
+* Update Kernel from `v5.4.16` to `v5.4.20` ([#898])
```
Comment (tjkirch) [inline CHANGELOG.md]:
```suggestion
* Update kernel to 5.4.20 ([#898])
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,98 @@
...
+* Improve SELinux policy ([#888])
```
Comment (tjkirch) [inline CHANGELOG.md]:
```suggestion
* Expand SELinux policy to include all classes and actions in 5.4 kernel ([#888])
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,98 @@
...
+* Include error messages in `apiserver` error responses ([#897])
+* Add `logdog`: a tool for collecting logs ([#880])
```
Comment (tjkirch) [inline CHANGELOG.md]:
```suggestion
* Add `logdog` to help users collect debug logs ([#880])
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,98 @@
...
+* Include objtool in `kernel-devel` ([#874])
```
Comment (tjkirch) [inline CHANGELOG.md]:
```suggestion
* Include objtool in `kernel-devel` for compiling external modules ([#874])
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,98 @@
...
+* Ignore termination signals in `updog` right before initiating reboot ([#869])
+* Pass `--containerd` flag to `kubelet` to specify `containerd` socket path ([#868])
+* Fix delay on reboot or power off ([#859])
+* Add `systemd.log_color=0` to remove ANSI color escapes from console log ([#836])
+* Reduce `containerd` logging when no errors have occurred ([#886])
+* Update Admin Container to `v0.5.0` ([#903])
```
Comment (tjkirch) [inline CHANGELOG.md]:
```suggestion
* Update admin container to v0.5.0 ([#903])
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,98 @@
...
+
+## Build changes
+
+* Set up GitHub Actions ([#837])
```
Comment (tjkirch) [inline CHANGELOG.md]:
```suggestion
* Set up GitHub Actions to test OS builds for PRs ([#837])
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,3 +1,98 @@
...
+* Update SDK to v0.10.1 ([#866])
+* Move built RPMs to build/packages ([#863])
```
Comment (tjkirch) [inline CHANGELOG.md]:
```suggestion
* Move built RPMs to `build/packages` ([#863])
```

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,3 +1,98 @@
...
+* Bump cargo-make to 0.30.0 ([#870])
+* Pass proxy environment variables through to docker containers. ([#871])
```
Comment (tjkirch) [inline CHANGELOG.md]:
```suggestion
* Pass proxy environment variables through to docker containers ([#871])
```

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,3 +1,98 @@
...
+* Third-party software package version bumps ([#895])
+  * libseccomp: bump version from `v2.4.2` to `v2.4.3` ([403bdeab])
+  * conntrack-tools: bump version from `v1.4.5` to `v1.4.6` ([06b9a942])
+  * libnftnl: bump version from `v1.1.5` to `v1.1.6` ([dfc5b5da])
+  * libnetfilter_conntrack: bump version from `v1.0.7` to `v1.0.8` ([54771d69])
+  * libxcrypt: bump version from `v4.4.15` to `v4.4.16` ([979249e5])
+  * strace: version bump from `v5.5` to `v5.6` ([63ddb075])
+* Update Rust dependencies ([#896])
+* Remove unused Rust dependencies ([#894])
+* Add temporary fix for arm64 in `coreutils` ([#879])
```
Comment (tjkirch) [inline CHANGELOG.md]:
```suggestion
* Add upstream fix for arm64 in `coreutils` ([#879])
```

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,3 +1,98 @@
...
+[#886]: https://github.com/bottlerocket-os/bottlerocket/pull/886
+[#888]: https://github.com/bottlerocket-os/bottlerocket/pull/888
+[#894]: https://github.com/bottlerocket-os/bottlerocket/pull/894
+[#895]: https://github.com/bottlerocket-os/bottlerocket/pull/895
+[#896]: https://github.com/bottlerocket-os/bottlerocket/pull/896
+[#897]: https://github.com/bottlerocket-os/bottlerocket/pull/897
+[#898]: https://github.com/bottlerocket-os/bottlerocket/pull/898
+[#899]: https://github.com/bottlerocket-os/bottlerocket/pull/899
+[#902]: https://github.com/bottlerocket-os/bottlerocket/pull/902
+[#903]: https://github.com/bottlerocket-os/bottlerocket/pull/903
```
Comment (tjkirch) [inline CHANGELOG.md:84]:
```suggestion
[#903]: https://github.com/bottlerocket-os/bottlerocket/pull/903
[73d120c9]: https://github.com/bottlerocket-os/bottlerocket/commit/73d120c9
```

File: CHANGELOG.md [hunk 13]
```diff
@@ -1,3 +1,98 @@
...
+* Add default wave files ([#881])
+
+## Documentation changes
+
+* QUICKSTART: Clarify which setup is optional ([#902])
+* QUICKSTART: add easier setup instructions using new eksctl release ([#849])
+* QUICKSTART: add note about allowing SSH access ([#839])
+* QUICKSTART: add section on finding AMIs through SSM parameters ([#838])
+* QUICKSTART: Add supported region list to QUICKSTART. ([73d120c9])
+* QUICKSTART: Add info about persistent volume csi plugin ([#899])
```
Comment (tjkirch) [inline CHANGELOG.md]:
```suggestion
* QUICKSTART: Add info about persistent volume CSI plugin ([#899])
```

File: CHANGELOG.md [hunk 14]
```diff
@@ -1,3 +1,98 @@
+# v0.3.2 (unreleased)
```
Comment (tjkirch) [inline CHANGELOG.md:1]:
```suggestion
# v0.3.2 (unreleased)

Special thanks to our first contributors, @inductor ([#853]), @smoser ([#871]), and @gliptak ([#870])!
```

File: CHANGELOG.md [hunk 15]
```diff
@@ -1,3 +1,98 @@
...
+
+## OS changes
+
+* Update Kernel from `v5.4.16` to `v5.4.20` ([#898])
+* Improve SELinux policy ([#888])
+* Include error messages in `apiserver` error responses ([#897])
+* Add `logdog`: a tool for collecting logs ([#880])
+* Include objtool in `kernel-devel` ([#874])
+* Ignore termination signals in `updog` right before initiating reboot ([#869])
+* Pass `--containerd` flag to `kubelet` to specify `containerd` socket path ([#868])
```
Comment (tjkirch) [inline CHANGELOG.md]:
```suggestion
* Pass `--containerd` flag to `kubelet` to specify `containerd` socket path, fixing some cAdvisor metrics ([#868])
```

File: CHANGELOG.md [hunk 16]
```diff
@@ -1,3 +1,89 @@
...
+* Add ability to add waves using TOML files ([#883])
+* Add default wave files ([#881])
+
+## Documentation changes
+
+* QUICKSTART: Clarify which setup is optional ([#902])
+* QUICKSTART: add easier setup instructions using new eksctl release ([#849])
+* QUICKSTART: add note about allowing SSH access ([#839])
+* QUICKSTART: add section on finding AMIs through SSM parameters ([#838])
+* QUICKSTART: Add supported region list to QUICKSTART. ([73d120c9])
```
Comment (tjkirch) [inline CHANGELOG.md]:
```suggestion
* QUICKSTART: Add supported region list ([73d120c9])
```

File: CHANGELOG.md [hunk 17]
```diff
@@ -1,3 +1,89 @@
+# v0.3.2 (unreleased)
+
+Special thanks to our first contributors, @inductor ([#853]), @smoser ([#871]), and @gliptak ([#870])!
+
+## OS changes
+
+* Update kernel to 5.4.20 ([#898])
+* Expand SELinux policy to include all classes and actions in 5.4 kernel ([#888])
+* Include error messages in `apiserver` error responses ([#897])
```
Comment (tjkirch) [inline CHANGELOG.md]:
```suggestion
* Include error messages in apiserver error responses ([#897])
```

File: CHANGELOG.md [hunk 18]
```diff
@@ -1,3 +1,89 @@
...
+* Add `logdog` to help users collect debug logs ([#880])
```
Comment (tjkirch) [inline CHANGELOG.md]:
```suggestion
* Add "logdog" to help users collect debug logs ([#880])
```

File: CHANGELOG.md [hunk 19]
```diff
@@ -1,3 +1,89 @@
...
+* Include objtool in `kernel-devel` for compiling external modules ([#874])
```
Comment (tjkirch) [inline CHANGELOG.md]:
```suggestion
* Include objtool in kernel-devel for compiling external modules ([#874])
```

File: CHANGELOG.md [hunk 20]
```diff
@@ -1,3 +1,89 @@
...
+* Ignore termination signals in `updog` right before initiating reboot ([#869])
```
Comment (tjkirch) [inline CHANGELOG.md]:
```suggestion
* Ignore termination signals in updog right before initiating reboot ([#869])
```

File: CHANGELOG.md [hunk 21]
```diff
@@ -1,3 +1,89 @@
...
+* Pass `--containerd` flag to `kubelet` to specify `containerd` socket path, fixing some cAdvisor metrics ([#868])
```
Comment (tjkirch) [inline CHANGELOG.md]:
```suggestion
* Pass `--containerd` flag to kubelet to specify containerd socket path, fixing some cAdvisor metrics ([#868])
```

File: CHANGELOG.md [hunk 22]
```diff
@@ -1,3 +1,89 @@
...
+* Fix delay on reboot or power off ([#859])
+* Add `systemd.log_color=0` to remove ANSI color escapes from console log ([#836])
+* Reduce `containerd` logging when no errors have occurred ([#886])
```
Comment (tjkirch) [inline CHANGELOG.md]:
```suggestion
* Reduce containerd logging when no errors have occurred ([#886])
```

File: CHANGELOG.md [hunk 23]
```diff
@@ -1,3 +1,89 @@
...
+* Set up GitHub Actions to test OS builds for PRs ([#837])
+* Update SDK to v0.10.1 ([#866])
+* Move built RPMs to `build/packages` ([#863])
+* Bump cargo-make to 0.30.0 ([#870])
+* Pass proxy environment variables through to docker containers ([#871])
+* Add parse-datetime crate ([#875])
+* Update third-party software packages ([#895])
+* Update Rust dependencies ([#896])
+* Remove unused Rust dependencies ([#894])
+* Add upstream fix for arm64 in `coreutils` ([#879])
```
Comment (tjkirch) [inline CHANGELOG.md]:
```suggestion
* Add upstream fix for arm64 in coreutils ([#879])
```

File: CHANGELOG.md [hunk 24]
```diff
@@ -1,3 +1,89 @@
...
+* QUICKSTART: add easier setup instructions using new eksctl release ([#849])
+* QUICKSTART: add note about allowing SSH access ([#839])
+* QUICKSTART: add section on finding AMIs through SSM parameters ([#838])
+* QUICKSTART: Add supported region list to QUICKSTART. ([73d120c9])
+* QUICKSTART: Add info about persistent volume CSI plugin ([#899])
+* QUICKSTART and README: Add appropriate ECR policy guidance ([#856])
+* README: Fix feedback link to point at existing section ([#833])
+* README: Add sentence about preview phase with feedback link ([#832])
+* README: Fixes and updates ([#831])
+* Update name of `early-boot-config` in API system diagram ([#840])
```
Comment (tjkirch) [inline CHANGELOG.md]:
```suggestion
* Update name of early-boot-config in API system diagram ([#840])
```

File: CHANGELOG.md [hunk 25]
```diff
@@ -1,3 +1,89 @@
...
+* Fix `updater` README's reference to data store version ([#844])
```
Comment (tjkirch) [inline CHANGELOG.md]:
```suggestion
* Fix updater README's reference to data store version ([#844])
```

**Review by tjkirch — APPROVED** [review]
Looks good, thanks.  I'm also suggesting we remove "`" code markers from tool names, because they're not "code" and there's so many of them that it's distracting in the overall changelog.

**Review by tjkirch — APPROVED** [review]

**Review by etungsten — APPROVED** [review]

**Pull request merged**